protocols/v2/binary-sv2/no-serde-sv2/codec/src/datatypes/copy_data_types.rs

```diff
@@ -1,4 +1,29 @@
-//! Copy data types
+/// Provides implementations for encoding and decoding copy data types as required by the SV2 protocol.
+/// Facilitates byte-level serialization and deserialization, particularly for types without dynamically-sized data.
+///
+/// ## Traits and Implementations
+///
+/// ### `Fixed`
+/// The `Fixed` trait is implemented for various data types to specify a fixed size for each, enabling consistent
+/// memory allocation during serialization. The `SIZE` constant for each type defines its byte size, with implementations
+/// provided for `bool`, unsigned integers (e.g., `u8`, `u16`, `u32`, `u64`), and custom types like `U24`.
+///
+/// ### `Sv2DataType`
+/// The `Sv2DataType` trait is implemented for these data types, providing methods for encoding and decoding operations
+/// such as `from_bytes_unchecked`, `from_vec_`, `from_reader_` (if `std` is available), and `to_slice_unchecked`.
+/// The methods use little-endian byte order for consistency across platforms.
+///
+/// ## Special Types
+///
+/// ### `U24`
+/// A custom 24-bit unsigned integer, represented as a `U24` struct, handles 3-byte data often used in SV2 protocols for
+/// memory-efficient encoding. Provides conversion methods to and from `u32`, with `TryFrom<u32>` ensuring values stay
+/// within the 24-bit range (0 to 16,777,215).
+///
+/// ## Macros
+/// The `impl_sv2_for_unsigned` macro streamlines the implementation of the `Sv2DataType` trait for unsigned integer types,
+/// ensuring little-endian byte ordering for serialization and handling both in-memory buffers and `std::io::Read`/`Write`
+/// interfaces when `std` is available.
```

Comment:
I cannot find a rendering for this header anywhere in the Rust Docs when I run `cargo doc`